verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -242,6 +242,15 @@ paths:
             minimum: 1
             maximum: 100
             default: 10
+        - name: changes_only
```

Comment:
nit but we haven't used _ as separator usually, we kept to camelCase for properties.

Reply:
I saw more uses of _ than camelCase in query parameters, e.g. access_token, validate_only, num_cases.

Reply:
oh, ok then.. I was thinking everything in our case data is camelCase, but request params it seems not... no big deal.